src/build.ts

```diff
@@ -248,29 +257,29 @@ function allDirs(pkgs: readonly Package[]) {
   return pkgs.reduce((a, p) => a.concat(p.dirs), [] as string[])
 }
 
-export async function build(main: string | readonly string[]) {
+export async function build(main: string | readonly string[], generateSourceMap = false) {
   let pkgs = typeof main == "string" ? [Package.get(main)] : main.map(Package.get)
-  let compiled = runTS(allDirs(pkgs), configFor(pkgs))
+  let compiled = runTS(allDirs(pkgs), configFor(pkgs, undefined, generateSourceMap));
   if (!compiled) return false
   for (let pkg of pkgs) {
-    await bundle(pkg, compiled)
+    await bundle(pkg, compiled, generateSourceMap);
     for (let file of pkg.tests.map(f => f.replace(/\.ts$/, ".js")))
       fs.writeFileSync(file, compiled.get(file))
   }
   return true
 }
 
-export function watch(mains: readonly string[], extra: readonly string[] = []) {
+export function watch(mains: readonly string[], extra: readonly string[] = [],generateSourceMap= false) {
```

Comment:
This could also take an options object as third argument. And please put spaces after commas and around operators.